marimo/_loggers.py

```diff
@@ -16,22 +16,26 @@
 _LOGGERS: dict[str, logging.Logger] = {}
 
 
+def log_level_string_to_int(level: str) -> int:
+    level = level.upper()
+    if level == "DEBUG":
+        return logging.DEBUG
+    elif level == "INFO":
+        return logging.INFO
+    elif level == "WARN":
```

Comment:
should we support `WARNING` too?

Reply:
i realize this just moving code, not new code, so no need to do this now

Reply:
Added WARNING. In a follow-up PR I will clean up what we're logging in the kernel